spryker/nginx/usr/local/share/env/25-framework

```diff
@@ -1,18 +1,46 @@
 #!/bin/bash
 
 export WEB_INCLUDES=${WEB_INCLUDES:-default-* yves-*}
-export ZED_WEB_INCLUDES=${ZED_WEB_INCLUDES:-default-01-site_ssl default-05-custom_scheme_flags default-05-site_healthcheck default-06-site_redirect_to_https default-10-base default-20-rewriteapp default-30-phpfpm default-50-disable_further_php zed-*}
+export ZED_WEB_INCLUDES=${ZED_WEB_INCLUDES:-default-01-site_ssl default-05-site_internal_requests default-05-custom_scheme_flags default-05-site_healthcheck default-06-site_redirect_to_https default-10-base default-20-rewriteapp default-30-phpfpm default-50-disable_further_php zed-*}
 
 ZED_WEB_DEFAULT_SERVER="$(convert_to_boolean_string "${ZED_WEB_DEFAULT_SERVER:-false}")"
 export ZED_WEB_DEFAULT_SERVER
 
-export ZED_WEB_SERVER_NAME=${ZED_WEB_SERVER_NAME:-zed}
-
 export WEB_DIRECTORY=${WEB_DIRECTORY:-public}
 
 export DATABASE_HOST=${DATABASE_HOST:-postgres}
-export DATABASE_USER=${DATABASE_USER:-spryker_user}
-export DATABASE_PASSWORD=${DATABASE_PASSWORD:-spryker_password}
+export DATABASE_USER=${DATABASE_USER:-spryker}
+export DATABASE_PASSWORD=${DATABASE_PASSWORD:-spryker}
 export DATABASE_NAME=${DATABASE_NAME:-spryker}
 
 export SYMFONY_APP_ENDPOINT=${SYMFONY_APP_ENDPOINT:-/index.php}
+
+# We need development dependencies for "vendor/bin/console setup:deploy:prepare-propel"
+export COMPOSER_INSTALL_FLAGS=${COMPOSER_INSTALL_FLAGS:-"--no-interaction --optimize-autoloader"}
+
+export APPLICATION_ENV=${APPLICATION_ENV:-production}
+
+export DATABASE_PLATFORM=${DATABASE_PLATFORM:-postgres}
+
+export ELASTICSEARCH_HOST=${ELASTICSEARCH_HOST:-elasticsearch}
+export ELASTICSEARCH_HOST_PORT=${ELASTICSEARCH_HOST_PORT:-9200}
+export REDIS_HOST=${REDIS_HOST:-redis}
+export REDIS_HOST_PORT=${REDIS_HOST_PORT:-6379}
+export MAILCATCHER_HOST=${MAILCATCHER_HOST:-mailcatcher}
+export MAILCATCHER_HOST_PORT=${MAILCATCHER_HOST_PORT:-1080}
+
+export YVES_HOST=${YVES_HOST:-yves}
+export YVES_HOST_PROTOCOL=${YVES_HOST_PROTOCOL:-https://}
```

Comment:
Can we remove `://` from here? Technically that's not really part of the protocol.